aws/policy/application-services.yaml

```diff
@@ -134,6 +134,13 @@ Statement:
       - elasticfilesystem:DeleteMountTarget
       - kinesis:CreateStream
       - kinesis:DeleteStream
+      - glue:CreateJob
+      - glue:DeleteJob
+      - glue:UpdateJob
+      - glue:GetJob
+      - glue:GetTags
+      - glue:TagResource
```

Comment:
Tag actions operate on `arn:aws:glue:us-east-1:{{ aws_account_id }}:*` (https://docs.aws.amazon.com/glue/latest/webapi/API_GetTags.html) could you please move the tag-related actions into `AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees`?